frontend/packages/kubevirt-plugin/src/components/modals/uninstall-kubevirt-modal/uninstall-kubevirt-message.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { ExternalLink, FirehoseResult } from '@console/internal/components/utils';
+import { connect } from 'react-redux';
+import { FormGroup, TextInput } from '@patternfly/react-core';
+import { VMIKind, VMKind } from '../../../types/vm';
+import { getBasicID, getLoadedData } from '../../../utils';
+import * as _ from 'lodash';
```

Comment:
we usually put the `lodash` on top, near the react ...

Reply:
You're right. I forgot to double-check my imports. I pushed an update that deprecates this file. I checked the imports in the remaining file, but you should probably check behind me. =)